mirrorlist

```diff
@@ -92,6 +96,8 @@ Server = https://uk-mirror.chaotic.cx/$repo/$arch
 # Arizona
 # * By: ayulauffer <t.me/ayulauffer>
 Server = https://us-az-mirror.chaotic.cx/$repo/$arch
+# Florida
```

Comment:
```suggestion
# Florida
# * By: Alisson <t.me/alissonlauffer>
```

Reply:
Also worth changing line 15 to `# * By: Alisson <t.me/alissonlauffer> (São Paulo)` too.

Reply:
ok